packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -120,6 +122,8 @@ import { VideoBackgroundEffectsDependency } from '@internal/calling-component-bi
 import { CallSurvey, CallSurveyResponse } from '@azure/communication-calling';
 import { CallingSoundSubscriber } from './CallingSoundSubscriber';
 import { CallingSounds } from './CallAdapter';
+/* @conditional-compile-remove(DNS) */
+import { DeepNoiseSuppressionEffect } from '@azure/communication-calling-effects';
```

Comment:
How big is the file size for this? Wondering if this needs lazy loaded like the video background effects

Reply:
i wonder if we should just do that so we have better decoupling from the calling effects package?

Reply:
It is adding ~4MB to our bundle size, it is still under discussion if we want to enforce the dependency resolution or handle it internally. There is a task created separately to track it https://skype.visualstudio.com/SPOOL/_workitems/edit/3811815